terraform/modules/cds_export/role_cds.tf

```diff
@@ -0,0 +1,9 @@
+resource "aws_iam_role" "cds" {

```

Comment:
I think it's confusing to have this in a different file to the assume role policy document, and similarly for the policy attachment below, since almost always you want to see the associated policy content when you look at a role - can you combine them?

This would also prevent the "policy.tf" file naming from being confusing for not containing the "read and write objects" policy that we apply to the task role, although with the expansion of this module that section may want to be pulled out into it's own file now (which I'd think would be different from the file containing the cds bits) - maybe not worth it, I'll leave you to ponder that.

Reply:
Sure I think combined makes more sense come to think of it as well. It's worth pulling out the "read and write" policy imo too, so I'll make those changes 👍

Reply:
Done 👍